### ISSUE TITLE
About the training results

### ISSUE THREAD

**Issue by qwefvb** [LogosRoboticsGroup/SeerDrive#4]
Thank you for this wonderful sharing! I trained the model following your code, but the evaluation results are quite different from those in the paper, with an average score of only 65.98.
My configuration file is consistent with the one in your open-source code. Could it be that certain hyperparameter settings are causing this discrepancy?

<img width="2522" height="366" alt="Image" src="https://github.com/user-attachments/assets/deb0cb10-470b-4da6-8c84-820efc485a27" />

**Comment by Zbozhou** [comment]
Based on experience and the characteristics of NAVSIM, this metric deviation is unlikely to be caused by configuration issues; it typically indicates the presence of a bug. It is recommended to inspect whether the dataset was fully downloaded, whether any crashes occurred during training, whether the required files were properly preloaded as described in the README, and whether the correct checkpoint was used for evaluation.

**Comment by qwefvb** [comment]
@wangdongzhuo I haven't trained it again because I couldn't find which step I went wrong. If you have trained this project, please let me know your results and the specific steps, thank you!

**Comment by qwefvb** [comment]
> [@qwefvb](https://github.com/qwefvb) Did you retrain the entire model and achieve the PDM88.9 score mentioned in the paper?

But I evaluated the author's open-source checkpoint, and the score was 88.5, very close to the score in the paper.

**Comment by Zbozhou** [comment]
> > [@qwefvb](https://github.com/qwefvb) Did you retrain the entire model and achieve the PDM88.9 score mentioned in the paper?
> 
> But I evaluated the author's open-source checkpoint, and the score was 88.5, very close to the score in the paper.

The checkpoint of the "release" version showed a result of 88.88 on my machine. By the way, during the training process, I found that it was not the last checkpoint that achieved the best performance, but rather it was necessary to test the last 5 epochs to select the highest one.
